Tests/Tracking/migration/DataPipelineMigrationAssistantTests.swift

```diff
@@ -23,10 +24,7 @@ class DataPipelineMigrationAssistantTests: UnitTest {
         diGraph.override(value: dateUtilStub, forType: DateUtil.self)
         diGraph.override(value: backgroundQueueMock, forType: Queue.self)
 
-        let moduleConfig = DataPipelineConfigOptions.Factory.create(writeKey: "test")
-        let implementation = DataPipelineImplementation(diGraph: diGraphShared, moduleConfig: moduleConfig)
-
-        DataPipeline.setupSharedTestInstance(implementation: implementation, config: moduleConfig)
+        let implementation = DataPipeline.createAndSetSharedTestInstance(diGraphShared: diGraphShared, config: dataPipelineModuleConfig)
         customerIO = CustomerIO(implementation: implementation, diGraph: diGraph)
```

Comment:
Directly calling `DataPipeline.createAndSetSharedTestInstance` and `CustomerIO(implementation: implementation, diGraph: diGraph)` feels like boilerplate code to me. I was expecting that these 2 lines of code in this file would instead call a super method to initialize the `CustomerIO` instance. 

Is there a reason why this test class cannot do this?

Reply:
Thanks for the idea, added extension methods for this.